### PR TITLE
UI - feedback bug fixes and improvements

### PR DESCRIPTION
**Pull request by aniketkatkar97**
I worked on 

- [x] Loading issue in KPI widget on landing page.

Screenshot 2023-07-25 at 7 14 24 PM

- [x] Fixed redirection links for the notification box tabs

https://github.com/open-metadata/OpenMetadata/assets/51777795/a9ffa442-4c02-44a0-ad24-d83558cf2c76

- [x] Changed redirection behaviour after deletion of data assets, stay on the page after soft deletion and redirect to landing page after hard deleted.

https://github.com/open-metadata/OpenMetadata/assets/51777795/4b26b893-3485-404c-829a-90fe6189f318

- [x] Fixed version page layout issue for lengthy change texts

Screenshot 2023-07-25 at 7 18 53 PM

- [x] Fixed data model not restoring after clicking the restore button.

https://github.com/open-metadata/OpenMetadata/assets/51777795/842254b4-34d0-4c50-8ed9-3edd75de083d

- [x] Added loader while fetching permissions in entity summary panel

Screenshot 2023-07-25 at 7 21 00 PM

- [x] Fixed alerts page error placeholder alignment issue

Screenshot 2023-07-25 at 7 21 54 PM

- [x] Changed the dashboard and pipeline URL display method for better view on data asset details page 

Screenshot 2023-07-25 at 7 23 38 PM

#
### Type of change:

- [x] Bug fix
- [x] Improvement

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/developers/contribute) document.
- [ ] My PR title is `Fixes <issue-number>: <short explanation>`
- [ ] I have commented on my code, particularly in hard-to-understand areas. 
- [ ] For JSON Schema changes: I updated the migration scripts or explained why it is not needed.

### CODE REVIEW

File: openmetadata-ui/src/main/resources/ui/src/components/common/DeleteWidget/DeleteWidgetModal.tsx [hunk 1]
```diff
@@ -150,8 +151,22 @@ const DeleteWidgetModal = ({
...
+            //    Example: We can still show the users list after deleting one user from the table in the UsersList page
+            // Else redirect to the landing page.
+            if (
+              afterDeleteAction &&
+              (shouldRunActionForDataAsset || !isDataAsset)
+            ) {
+              // Setting timeout for a window to show the success toast
+              setTimeout(() => {
+                afterDeleteAction();
+              }, 1000);
```
Comment (Sachin-chaurasiya) [inline openmetadata-ui/src/main/resources/ui/src/components/common/DeleteWidget/DeleteWidgetModal.tsx:169]: @aniketkatkar97 , why we need timeout here?

Comment (aniketkatkar97) [reply openmetadata-ui/src/main/resources/ui/src/components/common/DeleteWidget/DeleteWidgetModal.tsx:169]: @Sachin-chaurasiya For data assets page we passing refreshPage as afterDeleteAction, so the success toast does not appear to let user know about the result. I've added the comment for the same above that

Comment (Sachin-chaurasiya) [reply openmetadata-ui/src/main/resources/ui/src/components/common/DeleteWidget/DeleteWidgetModal.tsx:169]: The comment is to show the notification.
```
// Setting timeout for a window to show the success toast
```

File: openmetadata-ui/src/main/resources/ui/src/components/DataAssets/DataAssetsHeader/DataAssetsHeader.interface.ts [hunk 2]
```diff
@@ -75,6 +75,8 @@ export type DataAssetsHeaderProps = {
   permissions: OperationPermission;
   allowSoftDelete?: boolean;
   isRecursiveDelete?: boolean;
+  isDataAsset?: boolean;
```
Comment (chirag-madlani) [inline openmetadata-ui/src/main/resources/ui/src/components/DataAssets/DataAssetsHeader/DataAssetsHeader.interface.ts:78]: We don't need this prop at all

File: openmetadata-ui/src/main/resources/ui/src/components/common/DeleteWidget/DeleteWidgetModal.tsx (hunk 1 above)
Comment (chirag-madlani) [reply openmetadata-ui/src/main/resources/ui/src/components/common/DeleteWidget/DeleteWidgetModal.tsx:169]: I think we have added too many conditions here, @aniketkatkar97 Let's syncup for this tomorrow morning

Comment (aniketkatkar97) [reply openmetadata-ui/src/main/resources/ui/src/components/common/DeleteWidget/DeleteWidgetModal.tsx:169]: Sure

File: openmetadata-ui/src/main/resources/ui/src/utils/Assets/AssetsUtils.ts [hunk 3]
```diff
@@ -66,3 +67,14 @@ export const getEntityAPIfromSource = (
...
   }
 };
+
+export const handleDataAssetAfterDeleteAction = (isSoftDelete?: boolean) => {
+  if (isSoftDelete) {
+    setTimeout(() => {
+      refreshPage();
+    }, 1000);
+  } else {
+    window.history.pushState({}, '', '/');
```
[the remark this answers is not in the document]
Comment (chirag-madlani) [reply openmetadata-ui/src/main/resources/ui/src/utils/Assets/AssetsUtils.ts:77]: I prefer using it like: history.push('/')

**Review by Sachin-chaurasiya — DISMISSED** [review]
Looks Good 👍

File: openmetadata-ui/src/main/resources/ui/src/utils/Assets/AssetsUtils.ts [hunk 4]
```diff
@@ -66,3 +68,14 @@ export const getEntityAPIfromSource = (
...
 };
+
+export const handleDataAssetAfterDeleteAction = (isSoftDelete?: boolean) => {
+  if (isSoftDelete) {
+    setTimeout(() => {
+      refreshPage();
+    }, 1000);
+  } else {
+    history.push('/');
+    history.go(0);
```
Comment (chirag-madlani) [inline openmetadata-ui/src/main/resources/ui/src/utils/Assets/AssetsUtils.ts:79]: Do we need this extra history.go?

Comment (aniketkatkar97) [reply openmetadata-ui/src/main/resources/ui/src/utils/Assets/AssetsUtils.ts:79]: Pushing the path('/') just updates the url but does not update the page content, so triggering the refresh manually after that

File: openmetadata-ui/src/main/resources/ui/src/utils/Assets/AssetsUtils.ts [hunk 5]
```diff
@@ -66,3 +68,14 @@ export const getEntityAPIfromSource = (
       return getContainerByName;
   }
 };
+
+export const handleDataAssetAfterDeleteAction = (isSoftDelete?: boolean) => {
+  if (isSoftDelete) {
+    setTimeout(() => {
+      refreshPage();
```
Comment (Sachin-chaurasiya) [inline openmetadata-ui/src/main/resources/ui/src/utils/Assets/AssetsUtils.ts:75]: You can directly use `history.go(0)` here.

Comment (aniketkatkar97) [reply openmetadata-ui/src/main/resources/ui/src/utils/Assets/AssetsUtils.ts]: pushed the changes

**Pull request merged**